pkgs/applications/misc/sampler/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, buildGoModule, fetchFromGitHub, alsaLib }:
+
+buildGoModule rec {
+  pname = "sampler";
+  version = "1.0.3";
+
+  src = fetchFromGitHub {
+    owner = "sqshq";
+    repo = "sampler";
+    rev = "v${version}";
+    sha256 = "129vifb1y57vyqj9p23gq778jschndh2y2ingwvjz0a6lrm45vpf";
+  };
+
+  modSha256 = "0wgwnn50lrg6ix5ll2jdwi421wgqgsv4y9xd5hfj81kya3dxcbw0";
+
+  subPackages = [ "." ];
+
+  buildInputs = [ alsaLib ];
+
+  meta = with stdenv.lib; {
+    description = "Tool for shell commands execution, visualization and alerting";
+    homepage = "https://sampler.dev";
+    license = licenses.gpl3;
+    maintainers = with maintainers; [ uvnikita ];
```

Comment:
Do go modules usually have a platforms meta field?

Reply:
I've added `platforms = platforms.unix` since windows support seems to be experimental: https://github.com/sqshq/sampler#windows-experimental